live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/config/jackson/UpdateNodeMetadataEventSerializerTest.java

```diff
@@ -37,6 +37,7 @@
 import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
 
+import org.alfresco.hxi_connector.live_ingester.adapters.config.jackson.UpdateNodeMetadataEventSerializer;
```

Comment:
I think you must have changed the package of UpdateNodeMetadataEventSerializer but didn't update the test.

Reply:
Very good observation! Fixed